components/automate-chef-io/content/docs/saml.md

```diff
@@ -42,12 +42,53 @@ Users who sign in via SAML will have a session time of 24 hours before needing t
 
 ## Supported Identity Management Systems
 
+- [Azure AD]({{< relref "#azure-ad" >}})
 - Office365
 - OKTA
 - OneLogin
 - Ping
 - Tivoli Federated Identity Manager
 
+### Azure AD
+
+Using Azure AD as an SAML IdP requires specific configuration for both Azure AD and Chef Automate.
+
+{{< info >}}
+The signing certificate used for Chef Automate's SAML integration with Azure AD requires manual management.
+Signing key rotation is not done automatically.
+{{< /info >}}
+
+First, set up your configuration within Azure AD.
+Set up your application as a _"non-gallery application"_, and then configure its SAML log-in method.
+Enter `https://{{< example_fqdn "automate" >}}/dex/callback` as the value for both _Identifier (Entity ID)_ and _Reply URL (Assertion Consumer Service URL)_,.
```

Comment:
@srenatus Ideally, I would like to have a bulleted list with the Azure AD field & its expected value listed. Since I assume these on line 63 are the only two required fields and respective value, I wasn't going to change the format here.

Reply:
From playing with it, these are sufficient. Besides managing the certificate.

The original CFT docs (linked in https://github.com/chef/automate/issues/2887) included renaming the claims, but I've found that to be not necessary.

Reply:
The only thing I would add would to include `name_id_policy_format = "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress"` in the saml.toml. If I remember correctly, I had to do this to get Automate to use the email address as a login instead of the user UUID from AAD.

Reply:
Hrm I've tried it and it worked without. So, I'd rather keep this as simple as can be...? 🤔

Reply:
Cool. I set mine lab up several months ago so it's very possible some other thing got fixed between now and then. I'm good leaving it as simple as possible.